### ISSUE TITLE
g is not defined

### ISSUE THREAD

**Issue by B5656** [jsGanttImproved/jsgantt-improved#134]
Hello,
I try to make a Gantt chart with a function, but i get a :

"g is not defined"
 var vGantt = pGantt ? pGantt : g; //hack for backwards compatibility

Here is my code :

test.html :

<div style="position:relative; width:100%" class="gantt" id="GanttChartDIV"></div>
<script type="text/javascript" src="/js/test.js"></script>

test.js :

showGantt();
function showGantt(){
    
        var g = new JSGantt.GanttChart(document.getElementById('GanttChartDIV'), 'month');

        g.setOptions({
        vCaptionType: 'Complete',    
        vQuarterColWidth: 36,
        vDateTaskDisplayFormat: 'day dd month yyyy', 
        vDayMajorDateDisplayFormat: 'mon yyyy - Week ww',
        vWeekMinorDateDisplayFormat: 'dd mon', 
        vScrollTo : 'today',
        vShowTaskInfoLink: 1,
        vUseSingleCell: 1000,
        vFormatArr: ['Day', 'Week', 'Month', 'Quarter'], /
        });

        g.AddTaskItemObject({
        pID: 1,
        pName: "Task 1",
        pStart: "2018-09-05",
        pEnd: "2018-09-11",
        pLink:"",
        pClass: "gtaskgreen",
        pMile: 0,
        pComp: 100,
        pGroup: 0, 
        pParent: 0, 
        pOpen: 1, 
        pNotes: "",
        });

        g.Draw();

    }

**Desktop**
 - OS: Windows 7
 - Browser : Firefox 52.8.0

**Comment by sreemat1993** [comment]
@B5656 add property "pGantt:g" in the task item

**Comment by mariohmol** [comment]
Yes.. this is an issue.. i will try to change this in a matter that will not use this global g .. this still one of the issues to fix from 1.x version
